llvm/lib/CodeGen/AsmPrinter/AccelTable.cpp

```diff
@@ -251,7 +251,10 @@ class Dwarf5AccelTableWriter : public AccelTableWriter {
           const DWARF5AccelTableData &)>
           getIndexForEntry,
       bool IsSplitDwarf);
-
+  ~Dwarf5AccelTableWriter() {
+    for (DebugNamesAbbrev *Abbrev : AbbreviationsVector)
+      Abbrev->~DebugNamesAbbrev();
+  }
```

Comment:
Can we just remove the allocator instead and use plain old unique pointers?

Reply:
(I'm not familiar with how the abbrev stuff is implemented, so can't speak as to how much time this allocator is saving)

Reply:
I am going to accept and land to fix the bot.
Please followup with @felipepiovezan .

BTW. `std::destroy` should work here